src/Symfony/Bridge/Twig/Extension/ProfilerExtension.php

```diff
@@ -21,18 +21,17 @@
  */
 final class ProfilerExtension extends BaseProfilerExtension
 {
-    private ?Stopwatch $stopwatch;
-
     /**
      * @var \SplObjectStorage<Profile, StopwatchEvent>
      */
     private \SplObjectStorage $events;
 
-    public function __construct(Profile $profile, ?Stopwatch $stopwatch = null)
-    {
+    public function __construct(
+        Profile $profile,
+        private ?Stopwatch $stopwatch = null,
+    ){
```

Comment:
```suggestion
    ) {
```

as pointed out by fabbot